libtransact/src/workload/command.rs

```diff
@@ -159,13 +197,13 @@ impl std::str::FromStr for Command {
         let command = command_parts.next().map(|s| s.to_lowercase());
         match command {
             Some(ref set) if set == "set" => {
-                let address = command_parts
```

Comment:
All of this parsing should now be replaced with just a `protobuf::parse_from_bytes`